modules/core/src/v2/coins/eos.ts

```diff
@@ -124,6 +152,12 @@ export class Eos extends BaseCoin {
     return true;
   }
 
+  /**
+   * Get URLs of some active public nodes
+   */
+  getPublicNodeUrls(): string[] {
+    return common.Environments[this.bitgo.env].eosNodeUrls;
```

Comment:
Please use `this.bitgo.getEnv()` instead of accessing a private property of the bitgo object.